submissions/ArthurGorbenko/HTML5-Form/index.html

```diff
@@ -0,0 +1,487 @@
+                  src="images/website-icon.png"
+                  alt="website logo"
+                  class="form-radio_choice_type_image"
+                />
+                <span><b>Website</b></span>
+              </span>
+            </label>
+            <input
+              type="radio"
+              name="select_document"
+              id="radio_documents"
+              class="visually__hidden"
+            />
+            <label for="radio_documents" class="form-radio_choice_type_label">
+              <span class="form-radio_choice_type_button">
+                <img
+                  src="images/documents-icon.png"
+                  alt="website logo"
+                  class="form-radio_choice_type_image"
+                />
+                <span><b> Documents</b></span>
+              </span>
+            </label>
+            <input
+              type="radio"
+              name="select_document"
+              id="radio_social_acc"
+              class="visually__hidden"
+            />
+            <label for="radio_social_acc" class="form-radio_choice_type_label">
+              <span class="form-radio_choice_type_button">
+                <img
+                  src="images/social-icon.png"
+                  alt="website logo"
+                  class="form-radio_choice_type_image"
+                />
+                <span><b>Social account</b></span>
+              </span>
+            </label>
+          </section>
+          <section class="form-upload">
+            <span class="form-sections_headers_text form-upload_header"
+              >Upload your documents</span
+            >
+            <label for="form-upload" class="button form-upload_button_label"
+              ><span class="form-upload_button_label_text"
+                >Upload a new file</span
+              >
+              <input
+                type="file"
+                name=""
+                id="form-upload"
+                class="visually__hidden"
+              />
+            </label>
+            <table class="form-upload_table">
+              <thead>
+                <th class="form-table_cell form-table_cell_headers">
+                  File name
+                </th>
+                <th class="form-table_cell form-table_cell_headers">
+                  File size
+                </th>
+                <th class="form-table_cell form-table_cell_headers">
+                  Number of words
+                </th>
+                <th class="form-table_cell form-table_cell_headers">remove</th>
+              </thead>
+
+              <tbody>
+                <tr class="form-upload_table_row">
+                  <td class="form-table_cell form-table_cell_file_name">
+                    somedocs.doc
+                  </td>
+                  <td class="form-table_cell">78Kb</td>
+                  <td class="form-table_cell">823</td>
+                  <td class="form-table_cell form-table_cell_remove"></td>
```

Comment:
Trash icon must be a button.
When you want to delete something you press the button.)